app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -31,31 +31,27 @@ class ScheduleAppointmentSheet : BottomSheetActivity() {
   }
 
   private val possibleDates = listOf(
-      // TODO: Convert this to a data class; move display text to strings.xml
-      ("1 day" to (1 to ChronoUnit.DAYS)),
-      ("2 days" to (2 to ChronoUnit.DAYS)),
-      ("3 days" to (3 to ChronoUnit.DAYS)),
-      ("4 days" to (4 to ChronoUnit.DAYS)),
-      ("5 days" to (5 to ChronoUnit.DAYS)),
-      ("6 days" to (6 to ChronoUnit.DAYS)),
-      ("7 days" to (7 to ChronoUnit.DAYS)),
-      ("2 weeks" to (2 to ChronoUnit.WEEKS)),
-      ("3 weeks" to (3 to ChronoUnit.WEEKS)),
-      ("4 weeks" to (4 to ChronoUnit.WEEKS)),
-      ("5 weeks" to (5 to ChronoUnit.WEEKS)),
-      ("6 weeks" to (6 to ChronoUnit.WEEKS)),
-      ("7 weeks" to (7 to ChronoUnit.WEEKS)),
-      ("2 months" to (2 to ChronoUnit.MONTHS)),
-      ("3 months" to (3 to ChronoUnit.MONTHS)),
-      ("4 months" to (4 to ChronoUnit.MONTHS)),
-      ("5 months" to (5 to ChronoUnit.MONTHS)),
-      ("6 months" to (6 to ChronoUnit.MONTHS)),
-      ("7 months" to (7 to ChronoUnit.MONTHS)),
-      ("8 months" to (8 to ChronoUnit.MONTHS)),
-      ("9 months" to (9 to ChronoUnit.MONTHS)),
-      ("10 months" to (10 to ChronoUnit.MONTHS)),
-      ("11 months" to (11 to ChronoUnit.MONTHS)),
-      ("12 months" to (12 to ChronoUnit.MONTHS))
+      ScheduleAppointment("1 day", 1, ChronoUnit.DAYS),
```

Comment:
We should be moving these hardcoded strings to string resources. Maybe even look at building these dynamically using the `ChronoUnit` value. Thoughts?

Reply:
That is what we want to do. We have just deferred it for later.